### PR TITLE
br: fix check sum time inaccurate in restore log

### LINKED ISSUE #29158: br: Should print more valid message for checksum-duration
## Enhancement

As follow log from `br restore`, maybe it is no sense for "restore-checksum=1h57m24.639772777s" but [total-take=11m43.853700371s]. 
Should we print more valid message for checksum duration?

> [2021/10/21 17:08:09.280 +08:00] [INFO] [collector.go:66] ["Full restore success summary"] [total-ranges=1406] [ranges-succeed=1406] [ranges-failed=0] [split-region=18.662650702s] **[restore-checksum=1h57m24.639772777s]** [restore-ranges=1362] **[total-take=11m43.853700371s]** [restore-data-size(after-compressed)=43.71GB] [Size=43714987368] [BackupTS=428554737544069123] [total-kv=760000000] [total-kv-size=96.52GB] [average-speed=137.1MB/s]

### PR DESCRIPTION
**Pull request by fengou1**
### What problem does this PR solve?

Issue Number: close #29158

Problem Summary:
After restore data to tidb, the BR will do the checksum concurrently, we accumulate all check sum time of each go routines, as a result, the restore check sum time larger than total restore time.

### Check List

Tests 

- [x] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by YuJuncen** [comment]
Seems this would be the total running time of the checksum executor. This time would be similar to the `total-take` and I am afraid it's not really useful for the user.

Since we cannot record the accurate time of checksumming, maybe just remove the time?

**Comment by fengou1** [comment]
> Seems this would be the total running time of the checksum executor. This time would be similar to the `total-take` and I am afraid it's not really useful for the user.
> 
> Since we cannot record the accurate time of checksumming, maybe just remove the time?

it looks you are in point, remote this since we hard to get the sum of checksum time

**Review by YuJuncen — APPROVED** [review]

File: br/pkg/restore/client.go [hunk 1]
```diff
@@ -817,13 +818,8 @@ func (rc *Client) GoValidateChecksum(
 				if !ok {
 					return
 				}
+
 				workers.ApplyOnErrorGroup(eg, func() error {
-					start := time.Now()
-					defer func() {
-						elapsed := time.Since(start)
-						summary.CollectDuration("restore checksum", elapsed)
-						summary.CollectSuccessUnit("table checksum", 1, elapsed)
```
Comment (kennytm) [inline br/pkg/restore/client.go:825]: removing this also removes the "table checksum success count" though 🤔

Comment (fengou1) [reply br/pkg/restore/client.go:825]: it looks 
						summary.CollectSuccessUnit("table checksum", 1, elapsed)
removing the "table checksum success count" also.

no sure what else I can do?

Comment (fengou1) [reply br/pkg/restore/client.go:825]: we keep total checksum success unit.

**Review by kennytm — APPROVED** [review]

**Pull request merged**